catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -97,6 +99,17 @@
 
     private MetacardType metacardType = null;
 
+    private static final Map<String, String> MIME_TYPE_DATA_TYPE_MAP;
+
+    static {
+        MIME_TYPE_DATA_TYPE_MAP = new HashMap<>();
+        MIME_TYPE_DATA_TYPE_MAP.put("image", "Image");
+        MIME_TYPE_DATA_TYPE_MAP.put("audio", "Sound");
+        MIME_TYPE_DATA_TYPE_MAP.put("text", "Text");
+        MIME_TYPE_DATA_TYPE_MAP.put("message", "Text");
+        MIME_TYPE_DATA_TYPE_MAP.put("application", "Dataset");
```

Comment:
Application is a broad mime type that includes pdf, office, octet-stream, compressed files, etc.  This seems to be the most appropriate mapping given our domain of datatype